### DISCUSSION TITLE
[30 Giugno 2023] - Pipeline

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#20] [Pipeline]
### Traccia Esercizio
Si consideri l'architettura MIPS con pipeline mostrata nella figura in basso (e in allegato). Il programma qui di seguito somma i valori nella stringa, in valore assoluto. A fine procedura, il programma emette un warning se la stringa risulta negativa, e quindi stampa il risultato
<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/9fd1970b-5802-4963-9312-f5b5929bf3aa">
<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/4fb9c302-c2ed-41df-bb18-28d6a8760584">
<img width="867" alt="immagine" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/e4058718-4a82-4f9a-a35b-29b4116f0c33">

---

>**[Altre Soluzioni](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/85)**

**Comment by rimaout** [comment]
<img width="500" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/07005812-f47d-417b-a4b6-217c7a2eed0a">
<img width="500" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/8abf1e28-0a3a-466b-8f2d-7d8a61df1366">
<img width="500" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/bfe63471-fa13-47e5-8ef7-a1c7a2cead70">
<img width="500" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/a4a320c0-b4f4-4840-8426-29d90422b9af">

**Reply by alem1105** [reply]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/1203cd10-8521-48cd-9277-c8c7b318d8e6)
